core/modules/node/node.module

```diff
@@ -3696,3 +3731,72 @@ function node_language_delete($language) {
     ->condition('langcode', $language->langcode)
     ->execute();
 }
+
+/**
+ * Get a node from the tempstore.
+ *
+ * @param string $node_tempstore_id
+ *   The machine node_tempstore_id of the node item to load.
+ */
+function node_get_node_tempstore($node_tempstore_id) {
+  $caches = &backdrop_static(__FUNCTION__, array());
+  if (!isset($caches[$node_tempstore_id])) {
+    // Try loading from tempstore first to get in-progress changes.
+    $item = tempstore_get('node_tempstore', $node_tempstore_id);
+    $caches[$node_tempstore_id] = $item;
+  }
+
+  return $caches[$node_tempstore_id];
+}
+
+/**
+ * Store changes to a node or menu item in the temporary store.
+ *
+ * @param Node $item
+ *   The node item to save into tempstore.
+ * @param string $type
```

Comment:
params in docblock need to match actual params.

Reply:
Done